eng/psakefile.ps1

```diff
@@ -330,3 +361,33 @@ task run-examples -Depends init {
     
 }
 
+function Create-DocsEnv() {
+    param(
+        [string]
+        $EnvironmentPath,
+        [string]
+        $RequirementsPath,
+        [string[]]
+        $ArtifactPaths
+    )
+
+    Write-Host "##[info]Creating virtual environment for use with docs at $EnvironmentPath...";
+    python -m venv $EnvironmentPath
+
+    $activateScript = (Join-Path $EnvironmentPath "bin" "Activate.ps1");
+    if (-not (Test-Path $activateScript -ErrorAction SilentlyContinue)) {
+        Get-ChildItem $EnvironmentPath | Write-Host
+        throw "No activate script found for virtual environment at $EnvironmentPath; environment creation failed."
+    }
+
+    & $activateScript;
+    try {
+        pip install -r $RequirementsPath;
+        foreach ($artifact in $ArtifactPaths) {
+            pip install $artifact
+        }
+    }
+    finally {
+        deactivate;
```

Comment:
By the way there are other semicolons on lines 90, 374, 377, 385, 391, not sure if you want to remove those too (sorry, I didn't mention them in my other comment). It's a minor issue though.

Reply:
Thanks for the catch!